packages/aws-rfdk/lib/deadline/lib/stage.ts

```diff
@@ -154,6 +159,12 @@ export class Stage {
       throw new TypeError(`Expected a string "version" but got: ${typeof version}`);
     }
 
+    // Do minimum supported deadline version check
+    const minimumSupportedVersion = new Version([10, 1, 9, 2]);
+    if (minimumSupportedVersion.isGreaterThan(Version.parse(version))) {
```

Comment:
I feel it's a bit confusing to use a function called `isGreaterThan`, when the message says `is less than`.
You have to reread the code to understand what is really happening. 

Could we have both of these statements match? I guess `<` is used more often than `>`, so I would suggest changing the function to `isLessThan` and use
`if (manifestVersion.isLessThan(minimumSupportedVersion)) {`

What do you think?